OpenBoard.pro

```diff
@@ -438,14 +438,14 @@ linux-g++* {
     #LIBS += -lprofiler
     LIBS += -lX11
     LIBS += -lquazip5
-    INCLUDEPATH += "/usr/include/quazip"
+    INCLUDEPATH += "/usr/include/quazip5"
```

Comment:
(Disclaimer, I'm not a maintainer)

This is distribution specific, an actual fix needs to check multiple paths + check pkgconfig. I'd be in favor of using CMake for the project, since its better suited for finding external dependencies, but this is not plannend upstream as far as I know.

Reply:
Thanks, do you know if there is a way for me to have that patch in my repository so I can build the program myself but not included in the pull request?

Reply:
I'm facing this problem myself, since I have lots of local patches I need for OpenBoard to properly work on my "too new" system. So far I have worked in a branch containing those patches, and use `git rebase -i origin/${BRANCH}` in the end to throw out the unrelated changes before merging.

Reply:
I attempted this and github detected that I'd deleted the branch while I was trying to move it, and closed this pull request. So I've made a fresh pull request for the new branch. https://github.com/OpenBoard-org/OpenBoard/pull/433/files (Unfortunately there are still extraneous changes there too, but I don't want to have to create yet another pull request to fix it.)

What I did in my repository is to add a branch "build-fixes" with the extraneous changes on it, based on the "dev-qt5.1x" branch. I then used git cherry-pick to get rest of the changes, and made this branch "smart-import-build". Finally, I used "git cherry-pick build-fixes..smart-import-build" on a new "smart-import" branch to extract the commits I wanted in the pull request.